docs/src/main/paradox/client-side/client-transport.md

```diff
@@ -65,10 +65,10 @@ To make use of an HTTP proxy when using the `singleRequest` API you simply need
 the appropriate settings object when calling the single request method.
 
 Scala
-:  @@snip [HttpClientExampleSpec.scala]($test$/scala/docs/http/scaladsl/HttpClientExampleSpec.scala) { #https-proxy-example-single-request }
+:  @@snip [HttpClientExampleSpec.scala](/docs/src/test//scala/docs/http/scaladsl/HttpClientExampleSpec.scala) { #https-proxy-example-single-request }
```

Comment:
doubleslash is weird though?

Reply:
oops, most had leading slashes but some didn't. Updated to make this consistent.